Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -65,7 +65,7 @@
         prob: 0.5
       - id: ClothingEyesGlassesSecurity
       - id: ClothingHeadsetAltSecurity # WD
-      - id: ClothingHandsGlovesColorBlack
+      - id: ClothingHandsGlovesColorBlackSecurity # WD
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)